### PR TITLE
Improvement map display

### PR DESCRIPTION
**Pull request by teehamaral**
This PR includes an improvement on the map displaying in the U-Reporters and Polls pages. (#304)

For example, from:
screen shot 2017-08-24 at 13 55 32

To:
screen shot 2017-08-24 at 13 55 12

### CODE REVIEW

**Comment by teehamaral** [comment]
Can you review?

**Comment by teehamaral** [comment]
@norkans7 any thoughts?

File: templates/public/polls.haml [hunk 1]
```diff
@@ -354,6 +354,9 @@
         $(this).addClass('active');
         $(this).children('a').removeClass('primary-color');
         $(this).children('a').addClass('primary-bg-color');
+
+        var currentId = $(this).context['hash'];
```
Comment (norkans7) [inline templates/public/polls.haml]: I guess this can be easier with CSS ".poll-map svg" selector

File: templates/public/ureporters.haml [hunk 2]
```diff
@@ -143,6 +143,7 @@
           });
 
           window.initMap('ureport-location-map-id', boundaries, ajaxUrl,  '{{ org|config:"district_label"}}', colorsList, '{{ org|config:"ward_label"}}', '{{reporters|intcomma}}');
+          $(this).find('svg').css('height', '480px');
```
Comment (norkans7) [inline templates/public/ureporters.haml]: same here .ureport-location-map svg selector in CSS

**Comment by norkans7** [comment]
Addressed the issue using simpler CSS in #315

**Pull request closed without merging**